### ISSUE TITLE
Dafault values for battery mode

### ISSUE THREAD

**Issue by stakaz** [jsalatas/plasma-pstate#53]
Hello, I have the following issue: 

when I start my notebook without AC power or detach it, the slider for maximum freq is going back to 41% and the turbo switch becomes disabled. 

I can re-enable it back and scale the max frequency to 100% but the frequency will not go over the 40% mark still. 

Only after `sudo cpupower frequency-set -u 3600GHz` it will reach the maximum. 

But after that I can scale the maximum frequency with the slider and it works. 

So basically the question here is, 

1. is it a bug and 
2. are there predefined values for the battery mode?

**Comment by jsalatas** [comment]
You are probably using some power management software, like TLP, powertop, or what ever. And this the correct behavior actually for the system to low its power consumption. So you probably want to look at this issue. This widget doesn't change the settings itself in any case other than user's input.
